digi/xbee/exception.py

```diff
@@ -69,16 +76,18 @@ class XBeeDeviceException(XBeeException):
 
 class InvalidConfigurationException(ConnectionException):
     """
-    This exception will be thrown when trying to open an interface with an 
-    invalid configuration.
+    This exception will be thrown when trying to open an interface with an invalid configuration.
```

Comment:
NP: Line too long.

Reply:
In general, the library code does not restrict the line length to 80 chars maximum. Anyway I will undo this change.